go/arrow/array/array.go

```diff
@@ -27,7 +27,7 @@ import (
 const (
 	// UnknownNullCount specifies the NullN should be calculated from the null bitmap buffer.
 	UnknownNullCount = -1
-	NullValueStr = "(null)"
+	NullValueStr     = "(null)"
```

Comment:
Can you add a docstring here since now this is getting used as the official string expected for appending a null if you are using `AppendValueFromString`, so we should give this a docstring for the documentation to explain this.

Reply:
Done in 92ed97ce5b7782443655a344c916f8668a688092